### PR TITLE
updated cli getting started docs

### PR DESCRIPTION
**Pull request by epinzur**
also updated load-data docs

### CODE REVIEW

File: docs-src/modules/getting-started/pages/hello-world-cli.adoc [hunk 1]
```diff
@@ -243,7 +230,7 @@ Here's a more sophisticated example that touches on many of the unique features
 let cadence = hourly()
 
 # Anything can be named and re-used
-let hourly_big_purchases = purchase
+let hourly_big_purchases = Purchase
```
Comment (epinzur) [inline docs-src/modules/getting-started/pages/hello-world-cli.adoc:233]: the query as written still errors after this change.  I think we should update it to something that doesn't error, but lets do that in a different PR.

```
state: STATE_FAILURE
config:
  dataTokenId: 05f3eeb3-5d7d-4cb9-8d25-8e8ec6b0a85a
analysis: {}
fenlDiagnostics:
  fenlDiagnostics:
  - severity: SEVERITY_ERROR
    code: E0006
    message: Unbound reference
    formatted: |+
      error[E0006]: Unbound reference
         --> Query:23:27
         |
      23 |   last_visit_region: last(Pageview.region)
         |                           ^^^^^^^^ No reference named 'Pageview'
         |
         = Nearest matches: 'cadence', 'Purchase', '$input', 'purchases_now', 'purchases_yesterday'

  numErrors: "1"
metrics: {}
requestDetails:
  requestId: 0ba4b3980e0d372703eca40e69c8fa45
```

Comment (epinzur) [inline docs-src/modules/getting-started/pages/hello-world-cli.adoc:233]: the query as written still errors after this change.  I think we should update it to something that doesn't error, but lets do that in a different PR.

```
state: STATE_FAILURE
config:
  dataTokenId: 05f3eeb3-5d7d-4cb9-8d25-8e8ec6b0a85a
analysis: {}
fenlDiagnostics:
  fenlDiagnostics:
  - severity: SEVERITY_ERROR
    code: E0006
    message: Unbound reference
    formatted: |+
      error[E0006]: Unbound reference
         --> Query:23:27
         |
      23 |   last_visit_region: last(Pageview.region)
         |                           ^^^^^^^^ No reference named 'Pageview'
         |
         = Nearest matches: 'cadence', 'Purchase', '$input', 'purchases_now', 'purchases_yesterday'

  numErrors: "1"
metrics: {}
requestDetails:
  requestId: 0ba4b3980e0d372703eca40e69c8fa45
```

File: docs-src/modules/getting-started/pages/hello-world-cli.adoc [hunk 2]
```diff
@@ -309,6 +296,16 @@ When you configure `--preview-rows` Kaskada stops processing at the end of a bat
...
+== Cleaning Up
+
+When you're done with this tutorial, you can delete the table you created in order to free up resources.  Note that this also deletes all of the data loaded into the table.
+
+[source,bash]
+----
+# Delete the Purchase table
+kaskada-cli table delete --table Purchase
+----
+
```
Comment (epinzur) [inline docs-src/modules/getting-started/pages/hello-world-cli.adoc:308]: cleanup was dropped in previous PR, but I think it should be included.  Most getting-started docs I've seen include help to reset your state to a "new" system.

File: docs-src/modules/ROOT/pages/loading-data.adoc [hunk 3]
```diff
@@ -42,10 +42,7 @@ CLI::
 +
 [source,bash]
 ----
-kaskada-cli load \
-    --table Purchase \
-    --file-path file:///path/to/file.parquet \
-    --file-type parquet
+kaskada-cli table load Purchase file:///path/to/file.parquet
```
Comment (kerinin) [inline docs-src/modules/ROOT/pages/loading-data.adoc:45]: Can we use the same notes about file type here as are used for the other sections?

File: docs-src/modules/ROOT/pages/loading-data.adoc [hunk 4]
```diff
@@ -95,11 +92,14 @@ CLI::
...
 ----
+
+[IMPORTANT]
+====
+The CLI client infers a file's format from the extension of the file's path.
+The `--type parquet` flag can also be used to force a file to be treated as CSV.
```
Comment (kerinin) [inline docs-src/modules/ROOT/pages/loading-data.adoc:101]: `...to be treated as Parquet`

**Review by kerinin — APPROVED** [review]
